torminal.js

```diff
@@ -3,7 +3,7 @@ String.prototype.replaceAll = function(search, replacement) {
     return target.split(search).join(replacement);
 };
 var url = '';
-var files = ["readme.md"];
+var files = ["README.md"];
```

Comment:
This needs to be removed.  This array is not what we will be using.

Reply:
What are you guys using?

Reply:
We will be using the built in HTML5 file system API.  This will allow us to save things on a person’s computer.  It’s better then having to program out the file system ourselves and use cookies for saving.

Reply:
OK